components/bal/language-field.js

```diff
@@ -0,0 +1,96 @@
+import PropTypes from 'prop-types'
+import {Pane, Button, SelectMenu, Tooltip, TrashIcon} from 'evergreen-ui'
+
+import AssistedTextField from '@/components/assisted-text-field'
+
+function LanguageField({field, index, selectedLanguages, onChange, onSelect, onDelete, isToponyme}) {
+  const languagesList = [
+    {label: 'Breton', value: 'bre', disabled: false},
+    {label: 'Basque', value: 'eus', disabled: false},
+    {label: 'Alsacien', value: 'asw', disabled: false},
+    {label: 'Corse', value: 'cos', disabled: false},
+    {label: 'Créole martiquais | guadeloupéen', value: 'gyn', disabled: false},
+    {label: 'Créole réunionais', value: 'rcf', disabled: false},
+    {label: 'Occitan', value: 'oci', disabled: false},
+  ]
+
+  for (const language of languagesList) {
+    for (const selected of selectedLanguages) {
+      if (selected.value === language.value) {
+        language.disabled = true
+      }
+    }
+  }
+
+  const detectLanguage = field => {
+    for (const language of languagesList) {
+      if (field.value === language.value) {
+        return language.label
+      }
+    }
+  }
+
+  const handlePlaceholder = field.label === '' && field.value ? `Nom ${isToponyme ? 'du toponyme' : 'de la voie'} en ${detectLanguage(field).toLowerCase()}` : `Nom ${isToponyme ? 'du toponyme' : 'de la voie'} en langue régionale`
+
```

Comment:
```suggestion 
  const handlePlaceholder = `Nom ${isToponyme ? 'du toponyme' : 'de la voie'} en ${field.label === '' && field.value ? `${detectLanguage(field).toLowerCase()}` : 'langue régionale'}`
```

J'aurais quand même tendance à privilégier un if ou une fonction